docs/quantized_ops.md

```diff
@@ -99,12 +99,12 @@ orig_model.linear = q_linear
 
 | Weight Quantization Type | Activation Quantization Type | Dtype | Supported |
 |---|---|---|---|
-| per-channel | N/A | W8A16 | Yes |
-| per-channel | N/A | W4A16 | Yes |
+| per-channel (sym/asym) | N/A | W8A16 | Yes |
```

Comment:
consider create a rst file under `https://github.com/pytorch/xla/tree/master/docs/source` so it will show up in https://pytorch.org/xla/master/index.html

Reply:
Thanks, let me do in the following PR.

Reply:
Tracking in https://github.com/pytorch/xla/issues/7650